analysis/src/nanoAnalysis.h

```diff
@@ -24,7 +12,16 @@
 #include "ElecScaleFactorEvaluator.h"
 #include "BTagWeightEvaluator.h"
 
-class nanoAnalysis {
+void Events::Loop(){}
```

Comment:
should implement this in events.h file

Reply:
i see what ur doing

Reply:
Yeah but still it’s not quite right... soon i think i should figure out better way

Reply:
i think its ok because u want to generate Events.h and leave it unchanged